modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/DDMFormTemplateContextProcessor.java

```diff
@@ -156,6 +157,8 @@ protected DDMFormField getDDMFormField(JSONObject jsonObject) {
 		setDDMFormFieldVisibilityExpression(
 			jsonObject.getString("visibilityExpression"), ddmFormField);
 
+		setDDMFormFieldCustomProperties(jsonObject, ddmFormField);
+
```

Comment:
@carolmariaabb, I understand that we should remove the line breaks from lines 159 and 161 due to previous reviews from Brian. Besides, I believe that we should make clear on the commit message that the call to `setDDMFormFieldCustomProperties` should be placed where it is although Brian will ask us to make this call following alphabetic order. By the way, should it be one of the last methods to be called, right?

Reply:
Hi @natocesarrego, now that you raised this question, I believe we can put it in alphabetical order because when we do `ddmFormField.setProperty` with an existing property, the new value will overwrite the old one. Let me do some tests to make sure it's okay.